mozregression/utils.py

```diff
@@ -162,7 +172,7 @@ def url_links(url, regex=None, auth=None):
     """
     Returns a list of links that can be found on a given web page.
     """
-    response = get_http_session().get(url, auth=auth)
+    response = retry_get(args=(url, auth))
```

Comment:
This is not working - I suppose you need something like retry_get(args=(url,), kwargs=dict(auth=auth)))